Realtime/src/commonMain/kotlin/io/github/jan/supabase/realtime/RealtimeExt.kt

```diff
@@ -87,20 +113,23 @@ inline fun <reified Data : Any> RealtimeChannel.postgresListDataFlow(
             when (it) {
                 is PostgresAction.Insert -> {
                     val data = it.decodeRecord<Data>()
-                    val key = primaryKey.producer(data)
+                    val key = primaryKeys.producer(data)
                     cache[key] = data
                 }
+
                 is PostgresAction.Update -> {
                     val data = it.decodeRecord<Data>()
-                    val key = primaryKey.producer(data)
+                    val key = primaryKeys.producer(data)
                     cache[key] = data
                 }
+
                 is PostgresAction.Delete -> {
                     cache.remove(
-                        it.oldRecord[primaryKey.columnName]?.jsonPrimitive?.content
+                        it.oldRecord[primaryKeys.columnName]?.jsonPrimitive?.content
```

Comment:
This can't work with multiple primary keys (the object you receive obviously hasn't a weird concatenated string as key)

Reply:
Ah, that makes sense. Thanks for the fix.